crates/key-server/key-server-config.yaml

```diff
@@ -4,23 +4,35 @@
 # network: Testnet
 # network: Mainnet
 # network: Devnet
+#   seal_package: # What deployment of the Seal package to use - see below
 # network: !Custom
+#   seal_package: # Optional: What deployment of the Seal package to use - see below
```

Comment:
this is a breaking change to the config file? need to also update sui-operation right?

Reply:
Yes, it is breaking for the config and I'll follow up with a PR for sui-operations.